pekko-sample-distributed-workers-scala/build.sbt

```diff
@@ -0,0 +1,27 @@
+name := "pekko-distributed-workers"
+
+version := "1.0"
+
+scalaVersion := "2.13.8"
+val pekkoVersion = "0.0.0+26617-325e2156-SNAPSHOT"
+
+// allow access to snapshots
+resolvers += "Apache Nexus Snapshots".at("https://repository.apache.org/content/groups/snapshots/")
+
+val cassandraPluginVersion = "1.0.6"
```

Comment:
this is wrong - I actually think we haven't released pekko-persistence-cassandra snapshots yet - maybe in the next few days - we've unblocked some issues with progress on this today - because we are not publishing snapshots for pekko-connectors (including pekko-connectors-cassandra)

Reply:
I'll log an issue for this tonight. Since build passed, I'll merge this PR. The broken project must be missing from CI yml.

Reply:
Ok, see comment below with suggested update.